_data/companies.yml

```diff
@@ -0,0 +1,803 @@
+  foundation: 2004
+  urls:
+    - https://fanyi.sogou.com/text
+    - https://fanyi.sogou.com/download/pc/introduce
+  acquired_by:
+    company_id: tencent
+
+- name: Sunda Systems
+  id: sunda-systems
+  type: [ mt-provider ]
+  location: Finland
+  urls:
+    - https://www.sunda.fi/en/company
+
+- name: Symbyonics
+  id: symbyonics
+  foundation: 2014
+  active: false
+  urls:
+    - https://www.linkedin.com/company/symbyonics-llc/
+
+- name: SYSTRAN
+  id: systran
+  type: [ mt-provider ]
+  location: France
+  foundation: 1968
+  urls:
+    - https://www.systransoft.com/
+    - https://www.crunchbase.com/organization/systran-9361
+
+- name: Tarjama
+  id: tarjama
+  type: [ mt-provider ]
+  location: United Arab Emirates
+  foundation: 2008
+  urls:
+    - https://www.tarjama.com
+    - https://www.crunchbase.com/organization/tarjama-and-ureed
+
+- name: Tauyou
+  id: tauyou
+  type: [ mt-provider ]
+  location: Spain
+  foundation: 2006
+  urls:
+    - https://www.tauyou.com
+    - https://www.crunchbase.com/organization/ta-with-you
+  acquired_by:
+    company_id: transperfect
+
+- name: Tencent
+  id: tencent
+  type: [ technology-provider ]
+  location: China
+  foundation: 1998
+  urls:
+    - https:///www.tencent.com/en-us
+    - https://www.crunchbase.com/organization/tencent
+
+- name: TextShuttle
+  id: textshuttle
+  type: [ mt-provider ]
+  location: Switzerland
+  foundation: 2009
+  urls:
+    - https://www.textshuttle.ai/en/
+    - https://www.crunchbase.com/organization/textshuttle
+
+- name: Tilde
+  id: tilde
+  type: [ mt-provider ]
+  location: Latvia
+  foundation: 1991
+  urls:
+    - https://www.tilde.com
+    - https://www.crunchbase.com/organization/tilde-6e11
+
+- name: Tmxmall
+  id: tmxmall
+  type: [ technology-provider ]
+  location: China
+  foundation: 2014
+  urls:
+    - https://www.tmxmall.com/
+    - https://www.crunchbase.com/organization/tmxmall
+
+- name: TOIN
+  id: toin
+  type: [ technology-provider ]
+  location: Japan
+  foundation: 1963
+  urls:
+    - https://to-in.com/en/service/mt
+    - https://www.crunchbase.com/organization/toin
+
+- name: Translated
+  id: translated
+  type: [ lsp ]
+  location: Italy
+  foundation: 1999
+  urls:
+    - https://translated.com
+    - https://www.crunchbase.com/organization/translated
+
+- name: TranslateMe
+  id: translateme
+  type: [ mt-provider ]
+  location: South Africa
+  foundation: 2016
+  urls:
+    - https://translateme.network/
+
+- name: Transperfect
+  id: transperfect
+  type: [ lsp ]
+  location: United States
+  foundation: 1992
+  urls:
+    - https://www.transperfect.com
+    - https://www.crunchbase.com/organization/transperfect
+
+- name: Ubiqus
+  id: ubiqus
+  type: [ lsp ]
+  location: United States
+  foundation: 1991
+  urls:
+    - https://www.ubiqus.com
+    - https://www.crunchbase.com/organization/ubiqus
+  acquired_by:
+    company: Acolad
+
+- name: Unbabel
+  id: unbabel
+  type: [ mt-provider, lsp ]
+  location: Portugal
```

Comment:
I believe it's technically the US.

Reply:
Only Portugal is mentioned in Crunchbase.

Reply:
Perhaps I should add both?

Reply:
I'm not against it, but it sounds like a lot of work, if we have to do it for everybody.